doc/opentracks-schema-1.0.xsd

```diff
@@ -29,4 +29,13 @@
             </xsd:documentation>
         </xsd:annotation>
     </xsd:element>
+    <xsd:element name="accuracy"
+        type="xsd:float">
+        <xsd:annotation>
+            <xsd:documentation xml:lang="en">
+                Accuracy in meters of the current TrackPoint.
```

Comment:
Can we add a link the link and the description (i.e., 68 percentile)?

https://developer.android.com/reference/android/location/Location#getAccuracy()